### ISSUE TITLE
OwnerMemberFly Not Working

### ISSUE THREAD

**Issue by dublindevil88** [lewysDavies/GriefPreventionFlags#1]
### Description
When using the default claim flag "ownermemberfly" it seems to be applied in /claimflags but does not effect the owner with flight when they go into their claims

### Steps to Reproduce
/setdefaultclaimflag ownermemberfly
<de op yourself/remove administrative gp/gpflags perms>
walk into claim, no flight notification and cannot fly

### Expected Behavior
/setdefaultclaimflag ownermemberfly
owner of claim walks into their claim and is allowed to fly
owner of claim can leave their claim and the flight is disabled
if they /abandonclaim while flying in their claim, it gets disabled instead of allowing them to continue flying infinitely (issue with previous version of GPFlags)

### Server Information
* **Server version/platform:**        purpur -967
* **GriefPrevention Version:**        16.17.1-2b69191
* **GriefPreventionFlags Version:**   5.10.0 (downloaded from releases here)

**Comment by lewysDavies** [comment]
Can you confirm you do not have the permission gpflags.bypass.fly or gpflags.bypass and you're in survival mode. 

https://i.gyazo.com/41366be83994137f7d80033d05f8ddae.mp4

Seems to work for me?

**Comment by dublindevil88** [comment]
Currently using your latest version of gpflags (5.10.0) and gp (16.17.1-2b69191)
I cannot seem to make it work for some reason, and a video speaks a thousand words so here is the process for /setdefaultclaimflag and the issues I encountered. Note that before I did these tests I deleted the gpflags and the gpdata folders so there were no previous settings that could have affected the test.
https://vimeo.com/515156724

This one is a video where I used /gpflag rather than /setdefaultclaimflag. On the first login, I relogged while inside the claim and when I logged back in my flight was perma enabled. After relogging while not inside the claim and walking into it, I did not get the flight effect.
(cut it in the middle as there was a 1 minute login cooldown)
https://vimeo.com/515152329

If you are not encountering the issue then perhaps I am doing something wrong?

**Comment by dublindevil88** [comment]
bump

**Comment by lewysDavies** [comment]
Please confirm you are using Release v5.10.0_2

Tldr:
1. setdefaultclaimflag does not add OwnerMemberFly to all new claims
2. OwnerMemberFly keeps flight enabled on log out / in everywhere
